src/Authorization.AspNetCore/Authorization.AspNetCore.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup Label="Build">
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <AssemblyName>GraphQL.Server.Authorization.AspNetCore</AssemblyName>
+    <RootNamespace>GraphQL.Server.Authorization.AspNetCore</RootNamespace>
+    <Product>graphql-dotnet server</Product>
+    <Company>graphql-dotnet</Company>
+    <Authors>Pekka Heikura</Authors>
+    <Description>HTTP authorization middleware for graphql</Description>
+    <PackageProjectUrl>https://github.com/graphql-dotnet/server</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/graphql-dotnet/server</RepositoryUrl>
+    <RepositoryType>Git</RepositoryType>
+    <PackageTags>GraphQL authentication authorization middleware</PackageTags>
+    <Copyright>Pekka Heikura</Copyright>
+  </PropertyGroup>
+
+  <ItemGroup Label="Package References">
+    <PackageReference Include="GraphQL" Version="2.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Authorization" Version="2.1.0" />
```

Comment:
The other projects in this repository are currently referencing 2.0.x releases of the ASP.NET Core packages. I think this should be consistent with those.

Reply:
Done.